### ISSUE TITLE
Filter for Media Player within Chrome Browser

### ISSUE THREAD

**Issue by mcmjlm** [richardfrost/AdvancedProfanityFilter#412]
I want to filter videos within the Chrome browser that doesn't use a website to play the video.. is there a way to do this?

**Comment by richardfrost** [comment]
Hi @mcmjlm,

Are you trying to do something similar to #399? If so, that solution may work for you. Let me know if you have any other questions.

**Comment by mcmjlm** [comment]
no, not exactly. I'm trying to play movie files on my computer through google chrome so i can get the filtering on them.. am i going the wrong way to get this done?

**Comment by richardfrost** [comment]
So how are you playing the video? Just dragging it into a Chrome tab?
